game-core/src/main/java/games/strategy/engine/lobby/client/login/LoginPanel.java

```diff
@@ -27,15 +27,17 @@
   enum ReturnValue {
     CANCEL,
     LOGON,
```

Comment:
I know, not your fault but:
Shouldn't this be called LOGIN?

Reply:
Probably even LOGGED_IN maybe.

Reply:
Any intentions to change it as part of this PR or out of scope in your opinion?

Reply:
Out of scope, no need to change it in this branch